src/query/anonymization.c

```diff
@@ -471,6 +493,8 @@ static AnonymizationContext *make_query_anonymizing(Query *query, List *personal
     anon_context->expand_buckets = true;
   }
 
+  reject_aid_grouping(query, aid_refs);
```

Comment:
I pondered putting this under `validation.c`, however, firstly selecting AIDs doesn't make the query invalid in any sense, only pointless.

Secondly, under `validation.c` we'd need to have `personal_relations` handy, and I don't want to include it there without a strong reason.

Reply:
You don't need `personal_relations` to check for AID columns, look at `is_aid_column`, which reads the security label directly. I think this should be called by either `verify_anonymization_requirements` or by `compile_anonymizing_query`.